apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -10,7 +10,7 @@ import {
   AI_CUSTOMIZATIONS_LABELS,
 } from '../views/modelCustomization/constants';
 import {initialChatMessages} from '../constants';
-import {getChatCompletionMessage} from '../chatApi';
+import {getChatCompletionMessage} from '../../aiTutor/chatApi';
```

Comment:
Hey Emily - will you be calling on `getChatCompletionMessage` from this file? I see you have it in `aiTutorRedux.ts`. FWIW - I'm currently working on a PR that is updating this line.

Reply:
No -- AI Tutor doesn't need this. Your PR will be good to update it after this one.